packages/core/src/Account.ts

```diff
@@ -654,25 +658,20 @@ export class Account extends TypedEventEmitter<Events> {
       }
     });
 
-    const handleMissedNotifications = async (notificationId: string) => {
-      if (this.service?.mls) {
-        await this.service?.conversation.handleConversationsEpochMismatch();
-      }
-      return onMissedNotifications(notificationId);
-    };
```

Comment:
We don't need to eagerly rejoin conversation as soon as we detect missed messages. This process will be kicked off as soon as we receive (or send) a messages in the out-of-sync conversations

Reply:
The goal of this method is to run only if we've detected that we've missed messages (usually after 28 days). For each known conversation, we would then compare local epoch with the remote one and rejoin if necessary. Is this scenario still covered with this PR?